### PR TITLE
github implementation of CreatePR

### PR DESCRIPTION
**Pull request by michaelschiff**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: gitops/vcs/github/github.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	"errors"
+	"flag"
+	"golang.org/x/oauth2"
+	"github.com/google/go-github/v32/github"
+)
+
+var (
+	repoOwner = flag.String("github_repo_owner", "", "the owner user/organization to use for github api requests")
+	repo = flag.String("github_repo", "", "the repo to use for github api requests")
+	pat = flag.String("github_access_token", "", "the access token to authenticate requests")
```
Comment (kzadorozhny) [inline gitops/vcs/github/github.go:14]: I'd do  `os.Getenv("GITHUB_TOKEN")` as default value here. https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

Comment (michaelschiff) [reply gitops/vcs/github/github.go:14]: good idea

Comment (kzadorozhny) [reply gitops/vcs/github/github.go:14]: I think it is also good idea to add `github_api_endpoint` parameter that defaults to public GitHub.

**Review by kzadorozhny — DISMISSED** [review]
Thank you for the contribution!

**Review by kzadorozhny — DISMISSED** [review]

File: gitops/git/github/github.go [hunk 2]
```diff
@@ -42,7 +44,14 @@ func CreatePR(from, to, title string) error {
 		MaintainerCanModify: new(bool),
 		Draft:               new(bool),
 	}
-	_, _, err := gh.PullRequests.Create(ctx, *repoOwner, *repo, pr)
+	_, resp, err := gh.PullRequests.Create(ctx, *repoOwner, *repo, pr)
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+	log.Print("bitbucket response: ", string(body))
```
Comment (kzadorozhny) [inline gitops/git/github/github.go:50]: github response

**Review by kzadorozhny — APPROVED** [review]

**Pull request merged**